src/features/classification/Classification.scss

```diff
@@ -18,3 +18,12 @@
 .bdl-Classification-missingMessage {
     color: $bdl-gray-80;
 }
+
+.bdl-Classification-appliedByAiDetails {
+    margin-bottom: 20px;
+    margin-top: 3px;
```

Comment:
not using design tokens because these are legacy values to match the existing styling

Reply:
yes it's verified against the current mocks, this 20px is taken from the existing styling

Reply:
🤦‍♀️ my bad! there is a 20px token but not one for 3px <- I updated to use the closest spacing tokens and also confirm with our design & PM